src/components/MoneyRaisedDisplay.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import Proptypes from 'prop-types'
```

Comment:
For future reference, let's use a hook:
https://github.com/gladly-team/tab/blob/master/web/src/js/utils/hooks/useMoneyRaised.js